slices/tar.yaml

```
@@ -0,0 +1,31 @@
+package: tar
+
+essential:
+  - tar_copyright
+
+slices:
+  bins:
+    essential:
+      - tar_rmt
+      - tar_tar
+    contents:
+      /usr/sbin/tarcat:
+
+  copyright:
+    contents:
+      /usr/share/doc/tar/copyright:
+
+  rmt:
+    essential:
+      - libc6_libs
+    contents:
+      /etc/rmt: {symlink: /usr/sbin/rmt-tar}
```

```
lrwxrwxrwx root/root         0 2024-04-08 16:20 ./etc/rmt -> /usr/sbin/rmt
```

I see that `/etc/rmt` points to `/usr/sbin/rmt`. The latter is created via the postinst script. 

```sh
        update-alternatives --install /usr/sbin/rmt rmt /usr/sbin/rmt-tar 50 \
                --slave /usr/share/man/man8/rmt.8.gz rmt.8.gz \
                        /usr/share/man/man8/rmt-tar.8.gz
```

In my opinion, let's honour that chain.

```suggestion
      /etc/rmt:
      /usr/sbin/rmt: {symlink: /usr/sbin/rmt-tar}
```

Let me know what you think.

---

@rebornplusplus This is a forward-port of the existing tar SDF definition from 24.04 that is missing in 24.10.
Not sure I understand why we would make changes that cause it to diverge from what was landed originally in #320?

---

Yep, agreed @rebornplusplus 

@norrisjeremy is it something we should fix in 24.04.

---

fixed in https://github.com/canonical/chisel-releases/pull/507